cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-affected-records.cy.js

```diff
@@ -0,0 +1,92 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import TestTypes from '../../../support/dictionary/testTypes';
+import Permissions from '../../../support/dictionary/permissions';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import DevTeams from '../../../support/dictionary/devTeams';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import FileManager from '../../../support/utils/fileManager';
+import Users from '../../../support/fragments/users/users';
+import BulkEditActions from '../../../support/fragments/bulk-edit/bulk-edit-actions';
+import InventoryItems from '../../../support/fragments/inventory/item/inventoryItems';
+
+let user;
+const itemBarcodesFileName = `itemBarcodes_${getRandomPostfix()}.csv`;
+const previewFileName = `*-Updates-Preview-${itemBarcodesFileName}`;
+const barcode = `barcode-${getRandomPostfix()}`;
+const item = {
+  instanceName: `instanceName-${getRandomPostfix()}`,
+  firstBarcode: barcode,
+  secondBarcode: `secondBarcode_${barcode}`,
+  annexId: '53cf956f-c1df-410b-8bea-27f712cca7c0',
+};
+
+describe('bulk-edit', () => {
+  describe('in-app approach', () => {
+    before('create test data', () => {
+      cy.getAdminToken();
```

Comment:
cy.createTempUser already has       cy.getAdminToken();
should work without it

Reply:
I already fixed it!